### PR TITLE
More Dynamic wait tweaks.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
I messed up a step in the algo and made it so it was decreasing in its slowdown effect with more lag, not increasing as I had intended. This also had the side effect of making it kick in before the subsystem was causing lag.

I've also made the -1.5 magic number in the algo configurable. its was a simple buffer of cost the subsystem is allowed to use that isn't counted against it in dwait.

I also raised the default amount it scales up from 3 to 7.

Basically I'm trying to better increase the curve so its lower on the low end, and higher on the high end.

I has graph:

Y axis (up/down) is what dwait would be in ds, X axis is how long the subsystem is taking to process in ds.
(this is for the air subsystem)
**Red is old, Blue is new.**
Solid is average normal round.
Dotted line shows what would they would be if the singulo was free and actively eating/throwing items.

image

### CODE REVIEW

**Comment by Jalleo** [comment]
:+1: this has been causing the lag for people hasn't it?

**Comment by Midaychi** [comment]
u get an F for unlabeled Axis. Subsystem wait time per (Deciseconds per tick) vs Round time (minutes?)

**Comment by MrStonedOne** [comment]
>u get an F for unlabeled Axis. Subsystem fire time (Deciseconds per tick?) vs Round time (minutes?)

no!

wrong!

YOU'RE FUCKING WRONG!

I couldn't be assed to use a img editer but I did state what the axis are. read the shit before you comment.

**Comment by Midaychi** [comment]
Was just exaggerating the graph's lack for effect, ya dweeb.

**Comment by xxalpha** [comment]
I don't know what is going on, but that graph looks legit. :+1:

**Comment by Cheridan** [comment]
well it has a graph so i know its good
